File: src/com/kedzie/vbox/api/jaxb/IPciDeviceAttachment.java
package com.kedzie.vbox.api.jaxb;

public class IPciDeviceAttachment {

	protected String name;
	protected boolean isPhysicalDevice;
	protected int hostAddress;
	protected int guestAddress;

	public String getName() {
		return name;
	}

	public void setName(String value) {
		this.name = value;
	}

	public boolean isIsPhysicalDevice() {
		return isPhysicalDevice;
	}

	public void setIsPhysicalDevice(boolean value) {
		this.isPhysicalDevice = value;
	}

	public int getHostAddress() {
		return hostAddress;
	}

	public void setHostAddress(int value) {
		this.hostAddress = value;
	}

	public int getGuestAddress() {
		return guestAddress;
	}

	public void setGuestAddress(int value) {
		this.guestAddress = value;
	}

}

File: vbox-app/src/main/java/com/kedzie/vbox/api/jaxb/IPCIDeviceAttachment.kt
package com.kedzie.vbox.api.jaxb

import android.os.Parcelable
import com.kedzie.vbox.soap.KsoapObject
import kotlinx.android.parcel.Parcelize

@KsoapObject("IPCIDeviceAttachment")
@Parcelize
data class IPCIDeviceAttachment(
    var name: String,
    var isIsPhysicalDevice: Boolean,
    var hostAddress: Int,
    var guestAddress: Int) : Parcelable